Makefile

```diff
@@ -72,6 +72,9 @@ up/basic: up/essential
 .PHONY: up/essential
 up/essential: get vendor/hosts
 	@$(foreach SVC, $(START_ESSENTIAL), $(shell docker-compose -f services/$(SVC)/docker-compose.yml up -d))
+	@./vendor/neofs-adm --config services/ir/neofs-adm.yml morph init --alphabet-wallets ./services/ir --contracts vendor/contracts
+	@for f in $(shell ls ./services/storage/wallet*.json); do ./vendor/neofs-adm -c services/ir/neofs-adm.yml morph refill-gas --storage-wallet $${f} --gas 10.0 --alphabet-wallets services/ir; done 
```

Comment:
Can be just a glob without `$(shell ls` wrapping.

Also the output is 4 `Waiting for transactions to persist`, could you add some logging like (`Transfer GAS to wallet $${f}`)?

Reply:
Can you provide an example with glob, please? Not sure I understand what you mean.

Reply:
`@for f in $(shell ls ./services/storage/wallet*.json); do`
vs
`@for f in ./services/storage/wallet*.json; do`

Reply:
Done, thanks.